config/jobs/kubernetes/sig-release/cip/container-image-promoter.yaml

```diff
@@ -1,5 +1,30 @@
 presubmits:
   kubernetes/k8s.io:
+  - name: setup-k8sio-cip
```

Comment:
I think it's too early to add this for e2e testing. Let's flesh out more of the details before adding this job.

Reply:
I'm fine trying to land this job as `pull-container-image-promoter-lint` or something

Reply:
nit: I misread the repo, I would recommend setting this up against kubernetes-sigs/container-image-promoter, not against k8s.io